nautobot_chatops/dispatchers/__init__.py

```diff
@@ -3,7 +3,3 @@
 The classes implemented in this module implement a common API for "dispatching" content
 (plaintext messages, formatted text, UI elements, etc.) to the user of a given chat platform.
 """
-
-from .base import Dispatcher
-
-__all__ = ("Dispatcher",)
```

Comment:
Why was this needed to add a dev container?

Reply:
Fixed, reverted